apps/wing-console/console/app/tests/ex.redis/index.test.ts

```diff
@@ -1,10 +1,25 @@
+import { execSync } from "node:child_process";
+
 import { expect, test } from "@playwright/test";
 
 import { describe } from "../describe.js";
 import { getResourceNode } from "../helpers.js";
 
+const isDockerAvailable = () => {
+  try {
+    execSync("docker ps");
```

Comment:
Maybe `docker -v` would be better?

Reply:
we need to know if docker is running, not only if it is installed, so `docker -v` is not enough.